index.js

```diff
@@ -138,22 +143,34 @@ function getTemplates(viewId, data, structure){
             } else {
                 fileName = "template.docx";
             }
-            console.log("Template file name is " + fileName);
+            log.log("Template file name is " + fileName);
             file = templates[i].body;
             templateIdToFiles[templateIdsInOrder[i]] = {"file": file, "name": fileName};
         }
         //might need to write files to disk here? Not sure.
         var mergeData = mergeRecordsIntoTemplates(templatesToRecords, templateIdToFiles, structure)
         uploadMergeFiles(viewId, mergeData, templatesToRecords);
     }).catch(function(err) {
-       handleError(err);
+      checkFieldNames("template", config.template_table.view_id);
+      handleError(err);
     });
 }
 
 /**
+ * Checks to make sure that the viewId is a number
+ * @param {Number} viewId
+ */
+function checkTemplateViewId(viewId) {
+  if (typeof viewId != 'number') {
```

Comment:
If we want to support ids being strings as well, which I tend to think is a good idea, you can do:
`if (!isNaN(parseInt(viewId)) { ... }`